examples/misc_controls_map.html

```diff
@@ -84,7 +85,7 @@
 
 				const geometry = new THREE.BoxGeometry( 1, 1, 1 );
 				geometry.translate( 0, 0.5, 0 );
-				const material = new THREE.MeshPhongMaterial( { color: 0xffffff, flatShading: true } );
+				const material = new THREE.MeshPhongMaterial( { color: 0xeeeeee, flatShading: true } );
```

Comment:
This line is just an artistic change. Contrast was lower with the corrected color workflow, so I brought a bit of contrast back.